### DISCUSSION TITLE
desktop branch

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#114] [Q&A]
what is the desktop branch even for, i thought the desktop app is just the website as an app

also, it's waaayyyy too much work to manage two versions of the same thing, considering how many commits happen daily

**Comment by mariocraft987** [comment]
34

**Reply by raynepaws** [reply]
?

**Reply by mariocraft987** [reply]
commits daily

**Reply by raynepaws** [reply]
it's a lot
